gtest_parallel.py

```diff
@@ -754,9 +840,12 @@ def main():
   if options.timeout is not None:
     timeout = threading.Timer(options.timeout, sigint_handler.interrupt)
 
-  test_results = None
+  test_results = []
```

Comment:
I think we can have one sink instead of appending. Especially if you're OK with the suggested output format flag.